### DISCUSSION TITLE
I can't type in llama.cpp

### DISCUSSION THREAD

**Discussion by ghost** [ggml-org/llama.cpp#862] [General]
The program initializes, I get to

== Running in interactive mode. ==
 - Press Ctrl+C to interject at any time.
 - Press Return to return control to LLaMa.
 - If you want to submit another line, end your input in '\'.
 
 but then I can't type anything. I've tried two models, nothing. No matter what I press nothing comes up.

**Comment by BetaDoggo** [comment]
Are you seeing any disk activity while this is happening? I get the same thing until the model is finished loading into ram.

**Reply by ghost** [reply]
I do, so I just have to wait then?

**Reply by ivanstepanovftw** [reply]
Yeah, initial prompt is takes some time.

**Comment by YuxiangLiu-lyx** [comment]
I found that after finishing loading,  it automatically produces questions and answer itself,  what happen
